src/PrivateDns/PrivateDns/help/New-AzPrivateDnsVirtualNetworkLink.md

```diff
@@ -0,0 +1,235 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PrivateDns.dll-Help.xml
+Module Name: Az.PrivateDns
+ms.assetid: B78F3E8B-C7D2-458C-AB23-06F584FE97E0
+online version: https://docs.microsoft.com/en-us/powershell/module/az.privatedns/New-AzPrivateDnsVirtualNetworkLink
+schema: 2.0.0
+---
+
+# New-AzPrivateDnsVirtualNetworkLink
+
+## SYNOPSIS
+Creates a new private DNS virtual network link.
+
+## SYNTAX
+
+### VirtualNetworkId (Default)
+```
+New-AzPrivateDnsVirtualNetworkLink -ResourceGroupName <String> -ZoneName <String> -Name <String>
+ -VirtualNetworkId <String> [-EnableRegistration] [-Tag <Hashtable>] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Object
+```
+New-AzPrivateDnsVirtualNetworkLink -ResourceGroupName <String> -ZoneName <String> -Name <String>
+ -VirtualNetwork <VirtualNetwork> [-EnableRegistration] [-Tag <Hashtable>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzPrivateDnsVirtualNetworkLink** cmdlet creates a new private Domain Name System (DNS) virtual network link in the specified
+resource group and private zone. You must specify a unique link name for the *Name* parameter or the cmdlet will
+return an error. 
+You can use the *Confirm* parameter and $ConfirmPreference Windows PowerShell variable to control
+whether the cmdlet prompts you for confirmation.
+
+## EXAMPLES
+
+### Example 1: Create a Private DNS virtual network link
+```
+PS C:\>$Link = New-AzPrivateDnsVirtualNetworkLink -ZoneName "myzone.com" -ResourceGroupName "MyResourceGroup" -Name "mylink" -VirtualNetworkId "myvirtualnetwork" -EnableRegistration
```

Comment:
can you give an example of what a virtualnetworkid will actually look like? I don't think it's just a name.

Reply:
added a resource id.